fido-key-manager/src/main.rs

```diff
@@ -13,7 +13,6 @@ use std::time::Duration;
 use tokio_stream::StreamExt;
 #[cfg(feature = "solokey")]
 use webauthn_authenticator_rs::ctap2::SoloKeyAuthenticator;
-use webauthn_authenticator_rs::prelude::WebauthnCError;
```

Comment:
Gate on `#[cfg(feature = "solokey")]` to resolve the warning and not break the build